tests/test_select.py

```diff
@@ -142,3 +142,17 @@ def test_select_join_in_ansi89_syntax():
 
 def test_with_select():
     helper("WITH tab1 AS (SELECT 1) SELECT * FROM tab1")
+
+
+def test_with_select_one():
+    helper(
+        "WITH wtab1 AS (SELECT * FROM schema1.tab1) SELECT * FROM wtab1",
+        {"schema1.tab1"})
+
+def test_with_select_many():
+    helper(
+        "WITH wtab1 AS (SELECT * FROM schema1.tab1), "
+        "wtab2 AS (SELECT * FROM schema2.tab2)"
+        "SELECT * FROM wtab1 JOIN wtab2 ON wtab1.id = wtab2.id",
+        {"schema1.tab1", "schema2.tab2"}
+    )
```

Comment:
1. Change the testing SQL to the following. With column name specified, I believe it makes more sense for multiple CTE subclauses. FYI, I get this SQL from MySQL manual (https://dev.mysql.com/doc/refman/8.0/en/with.html). 
```sql
WITH
  cte1 AS (SELECT a, b FROM table1),
  cte2 AS (SELECT c, d FROM table2)
SELECT b, d FROM cte1 JOIN cte2
WHERE cte1.a = cte2.c;
```

2. Add a new line at end of file. It's always good coding style to end a file with newline.